### DISCUSSION TITLE
prebuilds

### DISCUSSION THREAD

**Discussion by MFAshby** [community/community#5319] [Codespaces] [answered]
Hi! 

A recent github blog post mentions 'prebuilds', i.e. codespaces where the code has already been downloaded and built ready to begin development 

https://github.blog/2021-08-11-githubs-engineering-team-moved-codespaces/

is this something which might be coming to public codespaces soon?

Kind regards,
Martin

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @MFAshby, yes we are working on releasing a private preview for this soon (early-mid sept) to get customer feedback. If you are interested to participate, please send an email to codespaces@github.com and we can follow up on next steps.
